modules/exploits/linux/http/tr069_ntpserver_cmdinject.rb

```diff
@@ -0,0 +1,146 @@
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Eir D1000 Modem CWMP Exploit POC',
```

Comment:
Is PoC needed in name? I don't see a payload encoded in here, but may be I missed it (mobile git sucks)